cmd/arc/main.go

```diff
@@ -24,6 +24,8 @@ var (
 	stripComponents        int
 	continueOnError        bool
 	specifyFileType        string
+	showHelp bool
+	showVersion bool
```

Comment:
It looks like these two are not set up and formatted correctly.

Reply:
I am not doing another commit just to add some spaces or whatever.

Reply:
Well, I certainly wouldn’t approve a PR with incorrect formatting. Just run `go fmt` on the file and it will sort out the formatting for you. This sort of thing is important for readable and maintainable code.

Reply:
(I know I haven't been very active with maintaining this repo lately, but) Yes, asking for formatting improvements is a normal part of most pull requests in Go projects -- in fact, a lot of CI are configured to fail if `go fmt` is not run on the code first. Thanks for understanding!